netci.groovy

```diff
@@ -27,8 +27,8 @@ def branch = GithubBranchName
                 // shell (for unix scripting)
                 shell("ls")
 		
-		shell("python tools/dotnet-bootstrap/base/lab/containers.py bake")
```

Comment:
With these commented out, what is the CI testing?  Can the CI just be removed for dotnet-core?

Reply:
I think we can leave it blank and doing the silent cross check for now. That way if there is something in the future, its easy to add. But I agree, it seems that this was the only reason we had CI.

Reply:
https://github.com/dotnet/core/issues/1068

Reply:
@Petermarcu has filed an issue to figure that out - https://github.com/dotnet/core/issues/1068

Reply:
Sounds good.  Maybe just delete these lines and call it good.

Reply:
I have a separate PR to empty out the netci.groovy - https://github.com/dotnet/core/pull/1069